StitchCore/Tests/stitch-ios-sdk-protoTests/Internal/Common/SynchronizedTests.swift

```diff
@@ -11,21 +12,25 @@ class SynchronizedTests: XCTestCase {
         var counter = Counter()
 
         let sema = DispatchSemaphore(value: 0)
-        Thread {
-            for i in 0 ..< 100 {
-                queue.addOperation {
-                    try! sync(self) {
-                        let value = counter.value
-                        if i % 2 == 0 {
-                            usleep(50)
+        if #available(OSX 10.12, *) {
```

Comment:
[q] Why was this added?

Reply:
It wasn't compiling without it but now when I remove it compiles so I'll just remove it.